integrations/pytorch/train.py

```diff
@@ -513,10 +526,18 @@ def train(
     :param loggers: List of loggers to use during training process
     """
     # loss setup
-    val_loss = utils.get_loss_wrapper(arch_key=train_args.arch_key, training=True)
+    val_loss = utils.get_loss_wrapper(
+        arch_key=train_args.arch_key,
+        loss_func_args=train_args.loss_func_args,
+        training=False,
```

Comment:
I additionally changed `training=False`, it seems to be a logical argument here.